### PR TITLE
[KARAF-5768] Add AIX platform detection when installing karaf as a sy…

### PR DESCRIPTION
**Pull request by baptistedaroit**
Add AIX platform detection when installing karaf as a system service

### CODE REVIEW

**Comment by baptistedaroit** [comment]
This PR just adds the AIX platform detection when using the `wrapper:install` command.

In combination with the PR #505, this will allow to succesfully integrate karaf as a service on AIX platform, as detailled below:

`AIX system detected :
  To install the service (and enable at system boot):
    $ ln -s /resgrp464/apache-karaf-4.2.1-SNAPSHOT/bin/karaf-service /etc/rc.d/init.d/

  To start the service when the machine is rebooted for all multi-user run levels
  and stopped for the halt, single-user and reboot runlevels:
    $ ln -s /etc/rc.d/init.d/karaf-service /etc/rc2.d/K20karaf-service
    $ ln -s /etc/rc.d/init.d/karaf-service /etc/rc2.d/S20karaf-service

    If your application makes use of other services, then you will need to make
    sure that your application is started after, and then shutdown before. This
    is done by controlling the startup/shutdown order by setting the right order
    value, which in this example it set to 20.

  To start the service:
    $ /etc/rc.d/init.d/karaf-service start

  To stop the service:
    $ /etc/rc.d/init.d/karaf-service stop

  To uninstall the service :
    $ rm /etc/rc.d/init.d/karaf-service
    $ rm /etc/rc2.d/K20karaf-service
    $ rm /etc/rc2.d/S20karaf-service

$ ln -s /resgrp464/apache-karaf-4.2.1-SNAPSHOT/bin/karaf-service /etc/rc.d/init.d/
$ /etc/rc.d/init.d/karaf-service start
Starting karaf...
$ /etc/rc.d/init.d/karaf-service status
karaf is running (15532250).
$ /etc/rc.d/init.d/karaf-service stop
Stopping karaf...
Waiting for karafto exit...
Stopped karaf.`

**Comment by baptistedaroit** [comment]
This also updates the Solaris specific instructions by:

- Adding the **Solaris/SunOS system detected** header 
- Fixing the two last instructions of how to uninstall the service (replace K with S to be coherent with what is indicated in the install section)

File: wrapper/src/main/java/org/apache/karaf/wrapper/commands/Install.java [hunk 1]
```diff
@@ -157,7 +157,7 @@ public Object execute() throws Exception {
                 System.out.println("    $ rm /etc/init.d/" + serviceFile.getName());
             } else {
 				System.out.println("");
-                System.out.println(INTENSITY_BOLD + "On Redhat/Fedora/CentOS Systems (SystemV):" + INTENSITY_NORMAL);
+                                System.out.println(INTENSITY_BOLD + "On Redhat/Fedora/CentOS Systems (SystemV):" + INTENSITY_NORMAL);
```
[the remark this answers is not in the document]
Comment (baptistedaroit) [reply wrapper/src/main/java/org/apache/karaf/wrapper/commands/Install.java]: Okay sorry. So how can I do to fix bad indentation when I catch one?

[the remark this answers is not in the document]
Comment (jbonofre) [reply wrapper/src/main/java/org/apache/karaf/wrapper/commands/Install.java]: Not a problem. If you want to fix code format, please do that in a dedicated commit.

[the remark this answers is not in the document]
Comment (baptistedaroit) [reply wrapper/src/main/java/org/apache/karaf/wrapper/commands/Install.java]: Thanks.

**Pull request merged**